IdokladSdk/Clients/Async/NotificationClient.Async.cs

```diff
@@ -11,5 +14,26 @@ public Task<ApiResult<bool>> DeleteAsync(int id, CancellationToken cancellationT
         {
             return DeleteAsync<bool>(id, cancellationToken);
         }
+
+        /// <summary>
+        /// Gives count of notifications with status New.
+        /// </summary>
+        /// <param name="cancellationToken">Cancellation token.</param>
+        /// <returns>Count of notifications with status New.</returns>
+        public Task<ApiResult<NewNotificationsCountGetModel>> GetNewNotificationCountAsync(CancellationToken cancellationToken = default)
```

Comment:
Chápu, že to tak máš v API, takže to tak nechej. Ale asi bych možná příště v tom API měl návratový typ jen ApiResult<int>.